src/iommu/vfio.c

```diff
@@ -436,6 +438,16 @@ static int vfio_get_device_fd(struct iommu_ctx *ctx, const char *bdf)
 	if (gfd < 0)
 		return -1;
 
+	vf_token = getenv("VFTOKEN");
```

Comment:
I missed in the last review, but can we have this VFTOKEN environment variable guide documentation to somewhere like `vfio_pci_open` API?  This API does not totally represent the vfio itself due to it might be about iommufd, it should be documented somewhere for users to give their own VFTOKEN.

Reply:
Thanks, fixed!